queue_entry.py

```diff
@@ -26,7 +26,6 @@
 import time
 import queue_model_objects_v1 as queue_model_objects
 import os
-import ShapeHistory as shape_history
```

Comment:
this can be replaced with:
cpos = queue_model_objects.CentredPosition(pos_dict)
I also came to this issue when working with qt4 and the fix is in master branch (like 654)
there are other places where is shape_history and that needs a cleaning...